design/resources/resources-design.json

```diff
@@ -1 +1,5 @@
-{"views":{"count_tags":{"map":"function (doc) {\n        for (var i = 0; i < doc.tags.length; i++) {\n          emit(doc.tags[i], 1);\n        }\n      }","reduce":"_count"},"titles":{"map":"function (doc) {\n        emit(doc.title.toLowerCase().trim(), doc.privateFor);\n      }"}}}
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
@chtelmko Why did you change this file at all?

I don't think this should have been changed, and now there are merge conflict markers which should always be handled before committing & pushing.

I will revert this file's changes.

Reply:
I didn't think there were anything changed on that file. I made that commit in order to create my pull request. I wasn't able to push my push my branch,It was saying I had uncommitted changes on the file. Thank you for reverting the changes.